publication/2025-06-16_announcing_ngxs_20/article.md

```diff
@@ -0,0 +1,162 @@
+
+You can find this feature documented in our [Lazy Loaded States](https://www.ngxs.io/concepts/state/lazy#lazyprovider) guide.
+
+```ts
+import { lazyProvider } from '@ngxs/store';
+
+const routes = [
+  {
+    path: 'invoices',
+    loadComponent: () =>
+      import('./invoices/invoices.component').then(m => m.InvoicesComponent),
+    canActivate: [
+      lazyProvider(async () => (await import('./states/invoices')).invoicesStateProvider)
+    ]
+  }
+];
+```
+
+In your state library, you can export a provider:
+
+```ts
+// states/invoices.ts
+export const invoicesStateProvider = provideStates([InvoicesState]);
+
+// OR using default export
+export default provideStates([InvoicesState]);
+```
+
+This approach reduces your initial bundle size and ensures that state providers are only loaded when they're actually needed.
+
+## Reduced RxJS Dependency
+
+NGXS v20 continues our efforts to reduce reliance on RxJS by:
+
+- Pulling fewer RxJS symbols ([PR #2309](https://github.com/ngxs/store/pull/2309), [PR #2310](https://github.com/ngxs/store/pull/2310))
+- Optimizing how we handle observable streams ([PR #2292](https://github.com/ngxs/store/pull/2292))
+
+These changes result in smaller bundle sizes and improved performance, especially for applications that don't heavily use RxJS elsewhere.
+
+## Bug Fixes
+
+We've addressed several important issues in this release:
+
+- Added root store initializer guard to prevent initialization issues ([PR #2278](https://github.com/ngxs/store/pull/2278))
+- Reduced change detection cycles with pending tasks ([PR #2280](https://github.com/ngxs/store/pull/2280))
+- Completed action results on destroy to prevent memory leaks ([PR #2282](https://github.com/ngxs/store/pull/2282))
+- Stopped contributing to stability once app is stable ([PR #2306](https://github.com/ngxs/store/pull/2306))
+
+## Plugin Improvements
+
+### Router Plugin
+
+- Changed `@Selector` to `createSelector` for better type safety and tree-shaking ([PR #2294](https://github.com/ngxs/store/pull/2294))
+
+### DevTools Plugin
+
+- Added `serialize` option for better control over state serialization ([PR #2337](https://github.com/ngxs/store/pull/2337))
+
+## Breaking Changes
+
+While we've worked to minimize breaking changes, there are a few changes to be aware of:
+
+- `executionStrategy` option has been removed, consider removing this option in your applications.
+- To get the `RouterState.state`, it is now required to call the state function — `select(RouterState.state<CustomRouterState>())`. Previously, it was possible to provide a generic type for the router state, e.g., `select(RouterState.state<CustomRouterState>)`, but this is not possible if state were a property.
```

Comment:
This sentence is a bit confusing to me. I don't understand what's the impact of the second part